src/components/ui/sliders.js

```diff
@@ -299,3 +299,137 @@ export const TestimonialCard = ({
     </div>
   );
 };
+
+/**
+ * @description - This component is combination of a list of images that when clicked will slide to the related slide in the slider.
+ * @param -
+ * @returns
```

Comment:
```suggestion
```
We shouldn't include these with no description. In this case, it's fine to remove them.